### PR TITLE
Add `getItems` endpoint for access to imported test items id

### PR DESCRIPTION
**Pull request by poyuki**
# [ADF-1052](https://oat-sa.atlassian.net/browse/ADF-1052)

## Summary
New endpoint provided for possibility to fetch items id for asynchronously imported tests

## How to test
- install branch
- use REST api endpoint to import test `/taoQtiTest/RestQtiTests/importDeferred`
- request test id by `/taoQtiTest/RestQtiTests/getStatus?id={taskId}`
- make request on new endpoint `/taoQtiTest/RestQtiTests/getItems?testUri={testId}`
- next payload should be provided
```json
{
    "success": true,
    "data": [
        "http://tao.local/mytao.rdf#xxxxxxxxxxxxx"
    ],
    "version": "{version}"
}
```

## TAE
https://oat-sa.atlassian.net/browse/ADF-1052?focusedCommentId=198991

### CODE REVIEW

File: actions/class.RestQtiTests.php [hunk 1]
```diff
@@ -161,6 +163,35 @@ public function importDeferred()
...
+
+            $this->returnSuccess(
+                array_map(static function (Resource $item) {
+                    return $item->getUri();
+                }, array_values($this->getQtiTestService()->getItems($testResource)))
+            );
+        } catch (MissingTestmodelException $e) {
+            $this->returnFailure(new common_exception_NotFound(
+                sprintf('Test %s not found', $testUri),
+                404,
```
Comment (kilatib) [inline actions/class.RestQtiTests.php:187]: Could we have constant here?

File: actions/class.RestQtiTests.php [hunk 2]
```diff
@@ -23,6 +23,8 @@
 use oat\taoQtiTest\helpers\QtiPackageExporter;
 use oat\taoQtiTest\models\tasks\ImportQtiTest;
 use oat\taoQtiItem\controller\AbstractRestQti;
+use core_kernel_classes_Resource as Resource;
+use oat\taoTests\models\MissingTestmodelException;
```
Comment (kilatib) [inline actions/class.RestQtiTests.php:27]: I see it has a bad readme file, could we add an example from PR description to the readme?

Comment (poyuki) [reply actions/class.RestQtiTests.php:27]: didn't get it. could you provide a bit more context to your comment?

Comment (poyuki) [reply actions/class.RestQtiTests.php:27]: if you mean request example i think in readme it covered by link to [OpenApi doc](https://github.com/oat-sa/extension-tao-testqti#rest-api)

Comment (kilatib) [reply actions/class.RestQtiTests.php:27]: Sorry, it nothing to related for that line I talked about README.md file

File: actions/class.RestQtiTests.php (hunk 1 above)
Comment (poyuki) [reply actions/class.RestQtiTests.php:187]: could you provide a bit more context here?

> Or fix link to SWAG

File: actions/class.RestQtiTests.php (hunk 2 above)
Comment (kilatib) [reply actions/class.RestQtiTests.php:27]: I just follow by link and README.md and somehow it wasn't. work for me

File: actions/class.RestQtiTests.php (hunk 1 above)
Comment (kilatib) [reply actions/class.RestQtiTests.php:187]: Sorry I jump in test window during review it for previous comment

File: actions/class.RestQtiTests.php (hunk 2 above)
Comment (poyuki) [reply actions/class.RestQtiTests.php:27]: just updated readme url by rawgithubcontent url

File: actions/class.RestQtiTests.php (hunk 1 above)
Comment (poyuki) [reply actions/class.RestQtiTests.php]: added constant usage from `slim/slim`

File: README.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Here is a list of known category options:
 REST API
 ========
 
-[QTI Test REST API](https://openapi.taotesting.com/viewer/?url=https://raw.githubusercontent.com/oat-sa/extension-tao-testqti/master/doc/swagger.json)
+[QTI Test REST API](https://raw.githubusercontent.com/oat-sa/extension-tao-testqti/master/doc/swagger.json)
```
Comment (wazelin) [inline README.md:31]:
```suggestion
[QTI Test REST API](https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Foat-sa%2Fextension-tao-testqti%2Fmaster%2Fdoc%2Fswagger.json)
```

**Review by wazelin — CHANGES_REQUESTED** [review]
This works very well, thank you!

One small concern, though – the response schema differs from the one mentioned in the ticket's description. Is it still OK?

**Comment by poyuki** [comment]
@wazelin, as i understand, for a moment there is no schema proposed in ticket. I can see only schema for `/taoQtiTest/RestQtiTests/getStatus` which was used as example and comprising for previous version of AC

**Comment by poyuki** [comment]
@wazelin response extend by array of objects

**Review by wazelin — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by kilatib — APPROVED** [review]

**Pull request merged**